WifiART/src/structs/LciState.java

```diff
@@ -17,59 +17,209 @@
 package structs;
 
 public class LciState {
+    // Constants
+    private static final int DEFAULT_LCI_VERSION = 1;
+    private static final AltitudeType DEFAULT_ALTITUDE_TYPE = AltitudeType.NO_KNOWN_ALTITUDE;
+    private static final MapDatum DEFAULT_MAP_DATUM = MapDatum.WGS84;
+
+
     // Parameters
+
+    private int lciVersion;
+
     private double latitude;
     private double latitudeUncertainty;
     private double longitude;
     private double longitudeUncertainty;
 
     private double altitude;
     private double altitudeUncertainty;
-    private String altitudeUnits; // "Meters" or "Floors"
-    private String altitudeDatum; // "WGS84", "NAD83 (NAVD88)", or "NAD83 (MLLW)".
+    private AltitudeType altitudeType; // Meters, Floors, or no altitude provided
+    private MapDatum mapDatum; // WGS84, NAD83 (NAVD88), or NAD83 (MLLW).
 
-    /* The regLocAgreement parameter is true if the STA is operating within a national
-       policy area or an international agreement area near a national border. */
+    /**
+     * The regLocAgreement parameter is true if the STA is operating within a national
+     *  policy area or an international agreement area near a national border.
+     */
     private boolean regLocAgreement;
```

Comment:
What is a "regLoc"?

Reply:
This terminology came out of the documentation (The documentation defines this field as "RegLoc Agreement", but it makes sense to spell this out for clarity. RegLoc is short for "Registered location". I think I should also explain what the DSE in "RegLoc DSE" stands for (DSE = Dependent STA enablement).

Reply:
Yes that would be great. The future reader of the code may not be that familiar with the spec so it would be better to spell this out.